packages/system/changelog.yml

```
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "2.1.0"
```

As this is a bug fix, Can we increment the patch version?
```suggestion
- version: "2.0.1"
```